Task07.cpp

```diff
@@ -0,0 +1,29 @@
+#include <iostream>
+
+void calculateHours(int& hour, int& minute, int startHour, int startMinute, int duration) {
+    startMinute += duration;
+    while(startMinute >= 60) {
+        startMinute -= 60;
+        startHour++;
+    }
+
+    if(startHour > hour) {
+        hour = startHour;
+        minute = startMinute;
+    }
+    else if(startHour == hour) {
+        if(startMinute > minute) minute = startMinute;
+    }
+}
+
+int main() {
+    int n, hour = 0, minute = 0;
+    std::cin >> n;
+    if(n < 0) return 0;
+    for(int i = 0; i < n; i++) {
+        int startingHour, startingMinute, duration;
+        std::cin >> startingHour >> startingMinute >> duration;
+        calculateHours(hour, minute, startingHour, startingMinute, duration);
+    }
+    std::cout << hour << " " << minute << std::endl;
```

Comment:
You do not check if hour is >= 24 when printing
Try the following input
4
19 15 120
20 20 25
18 25 230
21 10 3232